lib/puppet/functions/gitlab/cirunner/cmd_str.rb

```diff
@@ -0,0 +1,18 @@
+Puppet::Functions.create_function('gitlab::cirunner::cmd_str') do
+  dispatch :cmd_str do
+    param 'String', :name
+    param 'Array', :values
+    # only avalible in puppet 4.7
```

Comment:
I think Voxpupuli’s minimum supported puppet version is 4.10, so you should be fine setting the return type.